README.md

```diff
@@ -52,17 +52,22 @@ pipenv install django==2.1
 python manage.py makemigrations
 python manage.py migrate
 ```
-7. Run the server:
+7. Go to Users/keys_format.py
+- Rename file to keys.py.
+- Add the file name to gitignore.
```

Comment:
no need to add the file name to gitignore since it's already added there (keys.py)